src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form-search {
+  display: flex;
+  position: relative;
+  margin: 20px 0;
+}
+
+.form {
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #fff;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  outline: none;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input-field {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input-field-sm {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  line-height: 19.12px;
+  color: #3d4e61;
+}
+
+.form:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 1px 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form:focus {
+  font-weight: bold;
```

Comment:
placeholder font-weight should not change on focus
<img width="258" alt="image" src="https://user-images.githubusercontent.com/66749919/194618108-50d0f6e9-16b0-4a03-b0b2-5bd3056ca39b.png">